lib/splitclient-rb/clients/split_client.rb

```diff
@@ -62,22 +63,22 @@ def get_treatment(key, split_name, attributes = nil, split_data = nil, store_imp
       end
 
       start = Time.now
-      treatment_with_label = { label: Engine::Models::Label::EXCEPTION, treatment: Treatments::CONTROL }
+      treatment_label_change_number = { label: Engine::Models::Label::EXCEPTION, treatment: Treatments::CONTROL }
```

Comment:
So the one thing I'd call out is this:

how are you making sure that the right change number is reported even if there was an exception? Of course, if the exception was thrown while fetching the split (so that the split is null) then there can't be any change number.

I don't follow the code that well to be able to say that case is covered.